lib/FrontendTool/FrontendTool.cpp

```diff
@@ -1384,6 +1384,7 @@ static bool serializeSIB(SILModule *SM, const PrimarySpecificPaths &PSPs,
   serializationOpts.OutputPath = moduleOutputPath.c_str();
   serializationOpts.SerializeAllSIL = true;
   serializationOpts.IsSIB = true;
+  serializationOpts.SerializeOptions = true;
```

Comment:
Do you mean it's better to add new member like `IsLTO` instead of renaming and initializing `SerializeOptions`?

Reply:
If you only need the single state, that might be better than the entire serialization options.  Otherwise, renaming the variable is not really related to LTO, and if you want to rename it, you can do that in a separate change.